project_git_branch/README.rst

```diff
@@ -0,0 +1,63 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==================
+Project Git Branch
+==================
+
```

Comment:
add:
This modules extends the project object and add some information about the 
Git branch where the code is located